x/group/types.go

```diff
@@ -113,6 +108,85 @@ func (p *ThresholdDecisionPolicy) Validate(g GroupInfo) error {
 	return nil
 }
 
+// Implements DecisionPolicy Interface
+var _ DecisionPolicy = &PercentageDecisionPolicy{}
+
+// NewPercentageDecisionPolicy creates a new percentage DecisionPolicy
+func NewPercentageDecisionPolicy(percentage string, timeout time.Duration) DecisionPolicy {
+	return &PercentageDecisionPolicy{percentage, timeout}
+}
+
+func (p PercentageDecisionPolicy) ValidateBasic() error {
+	percentage, err := math.NewPositiveDecFromString(p.Percentage)
+	if err != nil {
+		return sdkerrors.Wrap(err, "percentage threshold")
+	}
+	if percentage.Cmp(math.NewDecFromInt64(1)) == 1 {
+		return sdkerrors.Wrap(errors.ErrInvalid, "percentage must be > 0 and <= 1")
+	}
+
+	timeout := p.Timeout
+	if timeout <= time.Nanosecond {
+		return sdkerrors.Wrap(errors.ErrInvalid, "timeout")
+	}
+	return nil
+}
+
+func (p *PercentageDecisionPolicy) Validate(g GroupInfo) error {
+	return nil
+}
+
+// Allow allows a proposal to pass when the tally of yes votes equals or exceeds the percentage threshold before the timeout.
+func (p PercentageDecisionPolicy) Allow(tally Tally, totalPower string, votingDuration time.Duration) (DecisionPolicyResult, error) {
```

Comment:
So with this decision policy, a no vote == a no_with_veto == an abstain vote, right? Is that desired? Should we maybe define a quorum of vetoes (e.g. 33%) that will make the result `Allow: false, Final: true`?